src/main/java/net/snowflake/client/core/HttpProtocol.java

```diff
@@ -0,0 +1,19 @@
+package net.snowflake.client.core;
+
+public enum HttpProtocol {
+
+    HTTP("http"),
+
+    HTTPS("https");
+
+    private final String protocol;
```

Comment:
we are not using the this field in any place so we can remove it and leave the enum in the simplest form:
```
public enum HttpProtocol {
    HTTP,
    HTTPS
}
```

Reply:
we ues it the `toString` which is used in `net.snowflake.client.core.HttpUtil#buildHttpClient` with: 
```java
key.getProxyHttpProtocol().toString()
```

Using toString is usually a code-smell. Should I add a method `String getProtocol()` method?

Reply:
`toString` does not seem to be good choice, especially that `HttpHost` constructor name the 3rd parameter (correctly) as `scheme` - it could be better approach for our enum to rename the field `protocol` in enum values to `scheme` and in `net.snowflake.client.core.HttpUtil#buildHttpClient` write `key.getProxyHttpProtocol().getScheme()`